### PR TITLE
[fixed] any props not known by DropdownMenu are passed through to the underlying ul

### PR DESCRIPTION
**Pull request by thealjey**

### CODE REVIEW

File: src/DropdownMenu.js [hunk 1]
```diff
@@ -80,37 +80,36 @@ class DropdownMenu extends React.Component {
...
     };
 
     let list = (
       <ul
-        className={classNames(this.props.className, classes)}
+        className={classNames(className, classes)}
         role="menu"
-        aria-labelledby={this.props.labelledBy}
+        aria-labelledby={labelledBy}
+        {...props}
```
Comment (jquense) [inline src/DropdownMenu.js]: I wouldn't pass the entire props object to the ul you will get issues where  things like id will be on two elements (this and the toggle). instead I would just explicitly apply the style prop like we do className

Comment (thealjey) [inline src/DropdownMenu.js]: I mean you're the boss and I only care about **`style`** right now, but the menu does not share anything with the toggle, or anything else for that matter, including its children.
Even **`DropdownButton`** is not passing any props down to **`Dropdown.Menu`**.

Comment (jquense) [inline src/DropdownMenu.js]: ahh yes, sorry, was mostly asleep when I commented, and thought it was on dropdown itself not the menu. yeah, this should be fine then

**Comment by taion** [comment]
LGTM

**Comment by jquense** [comment]
LGTM

**Comment by thealjey** [comment]
great, thanks!
how soon will this land on npm? :smile:

**Comment by jquense** [comment]
sometime this week I'd imagine. depends on when anyone of us get some time

**Pull request merged**